dotcom-rendering/src/components/TopBar.importable.tsx

```diff
@@ -137,6 +137,14 @@ export const TopBar = ({
 	return (
 		<Grid type="div" hasPageSkin={hasPageSkin}>
 			<div css={topBarStyles}>
+				<code
```

Comment:
I assume this would not be visible to users?

Reply:
This is debug code I added for testing on CODE, but will remove before merging. (As it _is_ visible in hot pink! 😅)